plugins/traffic/src/main/java/com/mapbox/mapboxsdk/plugins/traffic/TrafficPlugin.java

```diff
@@ -169,7 +189,19 @@ private void addLocalLayer() {
     );
 
     // #TODO https://github.com/mapbox/mapbox-plugins-android/issues/14
-    addTrafficLayersToMap(localCase, local, "bridge-motorway");
+    addTrafficLayersToMap(localCase, local, placeLayerBelow());
+  }
+
+  private String placeLayerBelow() {
+    if (belowLayer == null || belowLayer.isEmpty()) {
+      List<Layer> styleLayers = mapboxMap.getLayers();
+      for (int i = styleLayers.size() - 1; i >= 0; i--) {
+        if (!(styleLayers.get(i) instanceof SymbolLayer)) {
+          return styleLayers.get(i).getId();
+        }
+      }
+    }
+    return belowLayer;
```

Comment:
+  private String getIdLayerBelow() {
+    if (belowLayer == null || belowLayer.isEmpty()) {
+      List<Layer> styleLayers = mapboxMap.getLayers();
        Layer layer;
+      for (int i = styleLayers.size() - 1; i >= 0; i--) {
          layer = styleLayers.get(i);
+        if (!(layer instanceof SymbolLayer)) {
+          return layer.getId();
+        }
+      }
+    }
+    return belowLayer;